backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +175,30 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
```

Comment:
I might suggest removing IsAuthenticated, to allow this to be viewable by the public